README.md

```diff
@@ -56,16 +58,15 @@ See [the latest GitHub release](https://github.com/pingidentity/pingcli/releases
 OR
 
 Use the following single-line powershell command to install Ping CLI into '%LOCALAPPDATA%\Programs' directly.
-```text
-$latestReleaseUrl = Invoke-WebRequest -Uri "https://github.com/pingidentity/pingcli/releases/latest" -MaximumRedirection 0 -ErrorAction Ignore -UseBasicParsing -SkipHttpErrorCheck; `
```

Comment:
I'm concerned about the removal of `SkipHttpErrorCheck`. This makes the script possibly run into `302` redirect validation failures, even though this is expected.